tests/unit/test_protocols.py

```diff
@@ -123,18 +123,16 @@ def _compliance_tests(test_type=None):
         if full_path.endswith('.json'):
             for model, case, basename in _load_cases(full_path):
                 protocol = basename.replace('.json', '')
+                if _should_ignore_test(
+                    protocol,
+                    "input" if inp else "output",
+                    model['description'],
+                    case.get('id'),
+                ):
+                    continue
                 if 'params' in case and inp:
-                    if model.get('description') in PROTOCOL_TEST_BLACKLIST:
-                        continue
                     yield model, case, basename
                 elif 'response' in case and out:
-                    if _should_ignore_test(
-                        protocol,
-                        "output",
-                        model['description'],
-                        case.get('id'),
-                    ):
-                        continue
                     yield model, case, basename
```

Comment:
Nit: these two yield statements are identical, right?  Is there a reason to keep them in here?